readme.md

```diff
@@ -54,19 +58,20 @@ Visit [Azure IoT Dev Center][iot-dev-center] to learn more about developing appl
 
 ## Samples
 
-Samples are available at [Azure IoT Samples for C#](https://github.com/Azure-Samples/azure-iot-samples-csharp)
+Most of our samples are available at [Azure IoT Samples for C#](https://github.com/Azure-Samples/azure-iot-samples-csharp).
+
+If you are looking for a good device sample to get started with, please see the [device reconnection sample](https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/device/DeviceReconnectionSample).
+It shows how to connect a device, handle disconnect events, cases to handle when making calls, and when to re-initialize the `DeviceClient`.
 
 ## Contribute to the Azure IoT C# SDK
 
 If you would like to build or change the SDK source code, please follow the [devguide](doc/devguide.md).
 
 ## OS platforms and hardware compatibility
 
-> .NET MicroFramework will not be supported in future versions of the SDK.
-
 > .NET Standard 1.3 (IoT Hub SDKs only) is last supported in the [2020-02-27](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/2020-2-27) and in the [2020-1-31 LTS](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2020-1-31) releases.
 
-The IoT Hub device SDK for .NET can be used with a broad range of OS platforms and devices.
+The IoT Hub device SDK for .NET can be used with a broad range of OS platforms and devices, mainly modern versions Windows and Linux.
```

Comment:
The former statement was too vague. Even this is too vague, but it is a step in the right direction.